Runtime.md

```diff
@@ -0,0 +1,91 @@
+This guide will describe how GamesRunnr runs upkeep functions and how FSPAnalyzer is used to schedule them in GameStartr projects.
+
+### Table of Contents
+1. [GamesRunnr](#gamesrunnr)
+    1. [Games](#games)
+    2. [Interval and Speed](#interval-and-speed)
+    3. [Pause and Play](#pause-and-play)
+2. [FPSAnalyzer](#fpsanalyzer)
+
+# GamesRunnr
+
+GamesRunnr is a module for running a series of callbacks on timed intervals.
```

Comment:
It's a little unclear here whether the callbacks are all on their own individual intervals, or grouped into the same interval (run in order). It's the latter.